atlasdb-cassandra/build.gradle

```diff
@@ -100,20 +85,6 @@ dependencies {
   testCompileOnly 'org.immutables:value::annotations'
 }
 
-shadowJar {
-  mergeServiceFiles()
-  classifier ''
-
-  relocate('org.hibernate', 'shadowed')
```

Comment:
What all hibernate deps is this pulling in?

@pkoenig10  Can you please add a gradle dependency graph dump to the PR? Have you tested out  a local publish of this with one of your services? Does is bump any other transitive deps in any surprising ways?

Reply:
> Can you please add a gradle dependency graph dump to the PR?

The output of `./gradlew atlasdb-cassandra:dependencies` isn't interesting because it's exactly the same before and after this PR (module the removed shadow configurations).

It's the generated POM file that is different.

Regardless I've attached both below.

**`./gradlew atlasdb-cassandra:dependencies`**

[dependencies-before.txt](https://github.com/palantir/atlasdb/files/10214517/dependencies-before.txt)
[dependencies-after.txt](https://github.com/palantir/atlasdb/files/10214516/dependencies-after.txt)

**`./gradlew atlasdb-cassandra:generatePomFileForMavenPublication`**

[pom-before.xml.txt](https://github.com/palantir/atlasdb/files/10214521/pom-before.xml.txt)
[pom-after.xml.txt](https://github.com/palantir/atlasdb/files/10214522/pom-after.xml.txt)

Reply:
I've tested this out on a large internal project, link is below:

https://g.p.b/foundry/multipass/compare/atlasdb/6403

There are no change in transitive dependencies. The only notable change is that the `atlasdb-cassandra` dependencies are now being reflected in our GCV constraints. Notice that the constraint count has increased by 1 for each dependency in `atlasdb-cassandra`.

This means that is we now bump `libthrift`, this upgrade will be picked up by consumers.